components/SearchBox/components/index.module.css

```
@@ -0,0 +1,192 @@
+.searchBoxModalContainer {
+  @apply fixed
+    inset-0
+    z-50
+    flex
+    items-center
+    justify-center
+    bg-neutral-900
+    bg-opacity-90
+    dark:bg-neutral-900
+    dark:bg-opacity-90;
+}
+
+.searchBoxModalPanel {
+  @apply fixed
+    h-screen
+    w-full
+    bg-neutral-100
+    p-2
+    dark:bg-neutral-950
+    md:top-60
+    md:h-[450px]
+    md:max-w-3xl
+    md:rounded-xl
+    md:shadow-lg;
+}
+
+.searchBoxInnerPanel {
+  @apply pt-12
+    text-neutral-800
+  dark:text-neutral-400
+  md:p-2;
+}
+
+.searchBoxMagnifyingGlassIcon {
+  @apply absolute top-[10px] hidden h-6 w-6 md:block;
+}
+
+.searchBoxBackIconContainer {
+  @apply block md:hidden;
+}
+
+.searchBoxBackIcon {
+  @apply absolute top-[7px] block h-6 w-6 md:hidden;
+}
+
+.searchBoxInputContainer {
+  @apply relative px-2;
+}
+
+.searchBoxInput {
+  @apply w-full
+    rounded-b-none
+    border-b
+    border-neutral-300
+    bg-transparent
+    py-2
+    pl-8
+    pr-4
+    focus:outline-none
+  dark:border-neutral-900
+  dark:text-neutral-300
+  dark:placeholder-neutral-300;
```

```suggestion
    dark:border-neutral-900
    dark:text-neutral-300
    dark:placeholder-neutral-300;
```

indentation is missing

---

Yeah I'm having some problems with prettier, it doesn't really format CSS files from what I see. I'll take a look at that though

---

Prettier and other tools doesn't format `@apply` indentation.
So yeah we do it manually